package.json

```diff
@@ -52,10 +52,21 @@
       "git add"
     ]
   },
+  "jest": {
+    "snapshotSerializers": [
+      "./scripts/snapshot-serializers-ansi.js"
+    ],
+    "testMatch": [
+      "**/lib/**/?(*.)+(spec|test).[jt]s?(x)"
+    ]
+  },
   "dependencies": {
     "chalk": "^2.4.2",
     "convert-hrtime": "^3.0.0",
+    "cosmiconfig": "^5.2.1",
+    "fs-extra": "^8.1.0",
     "globby": "10.0.1",
+    "resolve-from": "^5.0.0",
```

Comment:
A `sindresorhus` module like require.resolve() but from a given path and will return undefined instead of throwing.